call-for-testing/action.yaml

```diff
@@ -45,13 +48,41 @@ runs:
       shell: bash
       run: |
         sudo snap install snapcraft --classic
+    
+    - name: Find the snapcraft.yaml path
+      id: yaml-path
+      shell: bash
+      run: |
+        if [[ -n "${{ inputs.snapcraft-yaml-path }}" ]]; then
```

Comment:
Did you also fix this space in the other workflows?

Reply:
In a future commit, yes!